noir-projects/aztec-nr/aztec/src/note/note_getter/test.nr

```diff
@@ -36,7 +36,7 @@ where
 }
 
 #[test]
-unconstrained fn processes_single_note() {
+unconstrained fn constrain_get_notes_processes_single_note() {
```

Comment:
It was not clear what the `note_getter` tests were testing so I clarified this now that we have some more getter tests under `test_environment::tests::notes`.